app/src/main/java/ch/epfl/sweng/erpa/services/UserService.java

```diff
@@ -0,0 +1,14 @@
+package ch.epfl.sweng.erpa.services;
+
+import com.annimon.stream.Optional;
+
+import java.util.Set;
+
+import ch.epfl.sweng.erpa.model.UserProfile;
+
+public interface UserService {
```

Comment:
Why do you need a precise interface? Can't the user classes just extends DataService<Game> ?